dependency/dependency_test.go

```diff
@@ -80,9 +75,79 @@ func TestMain(m *testing.M) {
 	exit := <-exitCh
 
 	testConsul.Stop()
+	testVault.Stop()
 	os.Exit(exit)
 }
 
+func runTestConsul() {
+	consul, err := testutil.NewTestServerConfig(
+		func(c *testutil.TestServerConfig) {
+			c.LogLevel = "warn"
+			c.Stdout = ioutil.Discard
+			c.Stderr = ioutil.Discard
+		})
+	if err != nil {
+		Fatalf("failed to start consul server: %v", err)
+	}
+	testConsul = consul
+}
+
+type vaultServer struct {
+	secrets_path string
```

Comment:
snake case -> camel case

Reply:
In this case I think you are right... cameCase is better for struct variables, particularly as this is referenced in other tests.